### ISSUE TITLE
The design of color palette

### ISSUE THREAD

**Issue by sainnhe** [sainnhe/gruvbox-material#11]
The color palette is still in beta stage, some colors may be adjusted in the future.

Currently, the colors look good on my device, but they may not look fine on yours. If you have any ideas or suggestions, please feel free to share them here.

**Note:** Since the colors display differently on different monitors, photos may be more helpful than screenshots if you want to upload images.

**Comment by sainnhe** [comment]
So I adjusted the color palette in [6994a16](https://github.com/sainnhe/gruvbox-material/commit/6994a16ca2626fe2933b407207e0ed052be71799).

This time I used some math tools to help me adjust the colors. In detail, I wrote a matlab script to calculate grayscale, and the colors have been tested on 3 devices. I'm personally very satisfied with current color palette.

Keep this issue open because there may still be color adjustments in the future.

Suggestions welcomed, especially if you are a professional designer.
